doc/rtd/development/index.rst

```diff
@@ -0,0 +1,48 @@
+Development
+***********
+
+``Cloud-init`` is an open source project that warmly welcomes community
+projects, contributions, suggestions, fixes and constructive feedback. If you
+would like to contribute to ``cloud-init``, this set of documentation will help
+orient you with our processes.
+
+-----
+
+Contributing
+============
+
+.. toctree::
+   :maxdepth: 1
+
+   contributing.rst
+   module_creation.rst
+   code_review.rst
+   dir_layout.rst
+
+Debugging
+=========
+
+.. toctree::
+   :maxdepth: 1
+
+   security.rst
+   analyze.rst
+   logging.rst
+   debugging.rst
+
+Testing
+=======
+
+.. toctree::
+   :maxdepth: 1
+
+   testing.rst
```

Comment:
Links named "Testing" and "Integration testing" back to back seems a bit awkward.

Reply:
"Testing" isn't a great name for the page - but I wasn't sure what to change it to that would make it more descriptive and useful. It seems sort of like an overview or introduction to testing? Should "Integration testing" be a subpage of it?

Reply:
> Should "Integration testing" be a subpage of it?

That's what I would suggest - integration testing is a subtopic of testing, and the testing page is more general so I think that would work.

Reply:
Perfect - I've changed that now :)